src/Acme/UserBundle/Entity/User.php~

```diff
@@ -0,0 +1,33 @@
+<?php
```

Comment:
What editor did you use? Some of them create backup files like this. You should add `*~` to your `.gitignore` so that you don't commit these.